gazebo/common/CMakeLists.txt

```diff
@@ -237,7 +229,7 @@ target_compile_definitions(gazebo_common
 target_link_libraries(gazebo_common
   PUBLIC
     ${Boost_LIBRARIES}
-    ${CURL_LIBRARIES}
+    CURL::libcurl
```

Comment:
Not something introduced in this PR, but it seems that `CURL` is not used in public headers, so probably `CURL::libcurl` can be linked as a `PRIVATE` linked library, instead of a `PUBLIC` one.

Reply:
Made PRIVATE in 9057ef9